internal/symbolizer/symbolizer.go

```diff
@@ -292,6 +292,16 @@ func demangleSingleFunction(fn *profile.Function, options []demangle.Option) {
 		fn.Name = demangled
 		return
 	}
+
+	// OSX has all the symbols prefixed with extra '_' so lets try
+	// once more without it
+	if strings.HasPrefix(fn.SystemName, "_") {
+		if demangled := demangle.Filter(fn.SystemName[1:], o...); demangled != fn.SystemName {
```

Comment:
Could we just pass `strings.StripPrefix(fn.SystemName, "_"), ...` in the `demangle.Filter` call above?

Reply:
Above? So just cut all the time no matter OS and everything? Then it won't work, since _ is usually used for mangled names for regular OS-es. And cutting it will simply break everything.

Reply:
Oh, got it.

Can we add a test that will exercise this code path?

Reply:
Ack. Added test case.